tests/Unit/Evolution/Systems/CurvedScalarWave/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(LIBRARY_SOURCES
   Test_PsiSquared.cpp
   Test_Tags.cpp
   Test_TimeDerivative.cpp
+  Worldtube/Test_PunctureField.cpp
```

Comment:
Should/could this get squash into the previous commit?

Reply:
whoops, meant to add the test in a separate commit. I squashed it all together.